crypto/x509/x509_vfy.c

```diff
@@ -237,11 +237,27 @@ static int verify_chain(X509_STORE_CTX *ctx)
     return ok;
 }
 
+int X509_STORE_CTX_verify(X509_STORE_CTX *ctx)
+{
+    if (ctx == NULL) {
+        ERR_raise(ERR_LIB_X509, ERR_R_PASSED_NULL_PARAMETER);
+        ctx->error = X509_V_ERR_INVALID_CALL;
```

Comment:
Assigning to `ctx->error` isn't going to work in this `if` statement.

Reply:
Indeed - also a little further down `X509_verify_cert()` - fixed.